src/ssl.c

```diff
@@ -21935,8 +21945,7 @@ unsigned long wolfSSL_ERR_peek_error_line_data(const char **file, int *line,
 }
 #endif
 
-#if defined(OPENSSL_EXTRA) || defined(OPENSSL_ALL) || \
-    defined(WOLFSSL_NGINX) || defined(WOLFSSL_HAPROXY)
+#if defined(OPENSSL_ALL) || defined(WOLFSSL_NGINX) || defined(WOLFSSL_HAPROXY)
```

Comment:
whoops, rebase error here.

This change needs to be reverted.  Caught it with `all-crypto-openssl-extra-coexist-smallstack` (got lucky!).

A recipe to repro is
```
./configure --enable-all --disable-all-osp --disable-opensslall --enable-opensslcoexist --with-sys-crypto-policy
```